src/test/java/advisor/authentication/SpotifyAccessTokenFetcherTest.java

```diff
@@ -79,12 +83,14 @@ void givenValidResponse_whenRequestingAccessToken_thenSuccessfulMessagesAndAcces
 
         // WHEN
         target.fetchAccessToken("myAccessCode");
-
         // THEN
         String expectedMessages = "making http request for access_token..." + System.lineSeparator()
                 + "response:" + System.lineSeparator()
                 + requestBody + System.lineSeparator();
-        assertThat(output).hasToString(expectedMessages);
+        String actualMessages = output.toString();
+        JsonElement actualJson = JsonParser.parseString(actualMessages.split("response:\n")[1].trim());
+        JsonElement expectedJson = JsonParser.parseString(expectedMessages.split("response:\n")[1].trim());
+        assertThat(actualJson).isEqualTo(expectedJson);
```

Comment:
Nice one. However I ran the test on Windows and it did not pass because of the line separator (`\r\n` instead of `\n`). 

What do you think of using this instead so that it works in any platform?

```java
        String actualMessage = output.toString();
        String responsePlaceholder = "response:" + System.lineSeparator();
        assertThat(actualMessage).contains("making http request for access_token..." + System.lineSeparator()
                + responsePlaceholder);
        JsonElement actualResponseBodyJson = JsonParser.parseString(actualMessage.split(responsePlaceholder)[1].trim());
        JsonElement expectedResponseBodyJson = JsonParser.parseString(requestBody);
        assertThat(actualResponseBodyJson).isEqualTo(expectedResponseBodyJson);
```

Reply:
Hello @carlos-velasco , thank you so much for the suggestion! I'll get them fixed!